week2/task3/Program.cs

```diff
@@ -0,0 +1,57 @@
+﻿var time = new JupiterTime(2, 20);
+var timeIn1Hours = time.AddHours(1);
+
+PrintTime(timeIn1Hours);
+
+void PrintTime(JupiterTime time)
+{
+    if (time.Minutes < 10) Console.WriteLine($"{time.Hours}:0{time.Minutes}");
+    else Console.WriteLine($"{time.Hours}:{time.Minutes}");
+}
+class JupiterTime
+{
+    private int _hours = 0, _minutes = 0;
+    public int Hours
+    {
+        get
+        {
+            return _hours;
+        }
+        set
+        {
+            if (value < 0) throw new Exception("negative number not acceptable");
```

Comment:
You should handle the negative hours like this.
if (value < 0)
_hours = (10 + _hours + value) % 10;
else if (value > 9)
_hours = (value % 10);
else
_hours = value;